modules/rhel-compute-requirements.adoc

```diff
@@ -19,7 +19,7 @@ ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.6 and later] with "Minimal" installation option.
+** Base operating system: Use link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.8 or a later version] with the "Minimal" installation option.
```

Comment:
```suggestion
** Base operating system: Use link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/index[{op-system-base} 8.8 or a later version] with the minimal installation option.
```

We avoid quotes in this instance as they could be overused in our docs and make other surrounding references less important.